client/src/main/java/com/vesoft/nebula/encoder/NebulaCodecImpl.java

```diff
@@ -98,6 +99,28 @@ private int getPartSize(String spaceName) throws RuntimeException {
             edgeItem.edge_type, edgeRank, dstId.getBytes());
     }
 
+    /**
+     * @param spaceName the space name
+     * @param srcId the src id
+     * @param edgeName the edge name
+     * @param edgeRank the ranking
+     * @param dstId the dst id
+     * @return
+     */
+    @Override
+    public byte[] inboundEdgeKey(String spaceName,
+                          String srcId,
+                          String edgeName,
+                          long edgeRank,
+                          String dstId)
+        throws RuntimeException {
+        int vidLen = getSpaceVidLen(spaceName);
+        int partitionId = getPartId(spaceName, srcId);
+        EdgeItem edgeItem = metaCache.getEdge(spaceName, edgeName);
+        return genEdgeKeyByDefaultVer(vidLen,partitionId, srcId.getBytes(),
```

Comment:
modify to `genEdgeKeyByDefaultVer(vidLen,partitionId, dstId.getBytes(), -edgeItem.edge_type, edgeRank, srcId.getBytes())`